cli/azd/internal/appdetect/java.go

```diff
@@ -137,6 +140,36 @@ func readMavenProject(filePath string) (*mavenProject, error) {
 	return &project, nil
 }
 
+func addDockerfileUnderProject(path string) error {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return fmt.Errorf("path does not exist: %s", path)
```

Comment:
Is it possible that it return other kinds of error? for example: permission denied.